src/App.js

```diff
@@ -1,24 +1,23 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import { Switch, Route, Link } from 'react-router-dom';
```

Comment:
Thanks and congratulations for making this pull request (PR). Everything Looks Good To Me (LGTM). I have managed to pull your branch to my local setup, installed dependencies and started the local server. Clicking the link at the bottom of the page updates the URL and renders the corresponding component. Good job. I have left a few, mostly trivial comments and questions below.

Reply:
It appears you have imported `Link` and `Logo` but they were never used in `App` component.  This has been caught by `eslint` on my setup.